### PR TITLE
direct data ingestion into Online store

### PR DESCRIPTION
**Pull request by vas28r13**
Signed-off-by: Vitaly Sergeyev <vsergeyev@better.com>

Issue docs: https://github.com/feast-dev/feast/issues/1863

Directly ingest data into the Online store.

Use Case
a model process/service will get a stream of data and would need to keep track of the new/updated features in the Online feature store

Code example
```python
feature_store_object.ingest_df("my_feature_view_name", dataframe_object)
```

**Process 1**
```python
# load feature store
feature_store = _get_feature_store()
for feature_data in _kafka_consume():
    # clean the data
    features_df = _clean_features(feature_data)

    # ingest the data directly into the online store
    feature_store.ingest_df("my_feature_view_name", features_df)
```

**Process 2 (materialization job)**
```python
feature_store = _get_feature_store()

# materialization from Offline sources
feature_store.materialize_incremental(end_date=datetime.now(timezone.utc))
```

### CODE REVIEW

File: sdk/python/feast/feature_store.py [hunk 1]
```diff
@@ -809,6 +809,26 @@ def tqdm_builder(length):
                 feature_view, self.project, start_date, end_date
             )
 
+    @log_exceptions_and_usage
+    def write_to_online_store(
```
Comment (benny-hal) [inline sdk/python/feast/feature_store.py:813]: @vas28r13 thx for adding this capability, we use this capability so we can write inference data in a real-time flow. However, the `get_feature_view` does not use the `allow_cache` option of the registry. This causes a lookup to refresh the registry for every call to `write_to_online_store`. This blocks the new capability from being used real-time. Would it make sense to add a PR that fixes that? thx...

Comment (adchia) [reply sdk/python/feast/feature_store.py:813]: would you mind making an issue so we can track this?

Comment (benny-hal) [reply sdk/python/feast/feature_store.py:813]: done

**Pull request merged**